angular-app/src/app/user-profile/user-profile.component.html

```diff
@@ -1,25 +1,32 @@
 <mat-toolbar>
   <span>Profile</span>
 </mat-toolbar>
-<mat-card class= "profile-card">
-    <mat-card-header>
-        <div mat-card-avatar></div>
-        <mat-card-title>John Doe</mat-card-title>
-        <mat-card-subtitle>johndoe</mat-card-subtitle>
-    </mat-card-header>
-    <mat-card-content>
-        <p>
-            Hello everyone! I love listening to all kinds of music but 
-            I play guitar and like to produce country tracks. Check out
-            my snippets, and if you want to collaborate, let's make
-            it happen!
-        </p>    
-    </mat-card-content>
-    <mat-card-actions>
-        <button mat-button>FOLLOW</button>
-        <button mat-button a routerLink= "/workspace">INVITE TO WORKSPACE</button>
-    </mat-card-actions>
-</mat-card>
+
+<mat-card class= "profile-card" *ngIf= "profile$ | async as profile">
+  <mat-card-header>
+    <img mat-card-avatar [src]="profile.photoUrl"/>
+    <mat-card-title>{{profile.displayName}}</mat-card-title>
+    <mat-card-subtitle>UID: {{profile.uid}}</mat-card-subtitle>
```

Comment:
Probably don't need to make the uid visible on your page.

Reply:
It's there for now because you need it to add collaborators and collaborator search isn't up yet :)